app/models/pending_induction_submission.rb

```diff
@@ -74,11 +74,26 @@ class PendingInductionSubmission < ApplicationRecord
 
   validate :start_date_after_qts_date, on: :register_ect
 
+  validate :no_future_induction_periods, if: -> { started_on.present? }, on: :register_ect
+
 private
 
   def start_date_after_qts_date
     return if trs_qts_awarded_on.blank?
 
     ensure_start_date_after_qts_date(trs_qts_awarded_on)
   end
+
+  def no_future_induction_periods
```

Comment:
General question. How has the team decided on when to move things out, in this case, into validators?